File: doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/api/model/Context.kt
package com.doordeck.multiplatform.sdk.api.model

import kotlinx.serialization.Serializable

object Context {

    @Serializable
    data class OperationContextData(
        val userId: String,
        val userCertificateChain: String,
        val userPublicKey: String,
        val userPrivateKey: String
    )
}

File: doordeck-sdk/src/mingwMain/resources/doordeck_headless_sdk/Model/Context.cs
namespace Doordeck.Headless.Sdk.Model
{
    public class OperationContextData
    {
        public string UserId { get; set; }
        public List<string> UserCertificateChain { get; set; }
        public string UserPublicKey { get; set; }
        public string UserPrivateKey { get; set; }

        public OperationContextData(string userId, List<string> userCertificateChain, string userPublicKey, string userPrivateKey)
        {
            UserId = userId;
            UserCertificateChain = userCertificateChain;
            UserPublicKey = userPublicKey;
            UserPrivateKey = userPrivateKey;
        }
    }
}